### PR TITLE
Enable VM Test and Use Latest Deb URL flag

### PR DESCRIPTION
**Pull request by chxchx**
Please do retest and only merge after https://github.com/istio/istio/pull/3993

also fixes https://github.com/istio/istio/issues/3995

### CODE REVIEW

File: prow/e2e-suite.sh [hunk 1]
```diff
@@ -66,6 +66,7 @@ trap cleanup EXIT
 # use uploaded yaml artifacts rather than the ones generated locally
 DAILY_BUILD=istio-$(echo ${ISTIOCTL_URL} | cut -d '/' -f 6)
 LINUX_DIST_URL=${ISTIOCTL_URL/istioctl/${DAILY_BUILD}-linux.tar.gz}
+DEB_URL=${ISTIOCTL_URL/istioctl/deb}
```
Comment (rkpagadala) [inline prow/e2e-suite.sh]: please add comment about this weird substitution thing.

Comment (chxchx) [reply prow/e2e-suite.sh]: I rebased from your change so we no longer have to do substitution

**Comment by chxchx** [comment]
The flag `--deb_url` is not yet included in the release candidate so tests are failing. Will do retest for the releases candidate the next day

**Pull request merged**